### PR TITLE
add imagemagick

### PR DESCRIPTION
**Pull request by ocefpaf**
There is a failure that I could not figure out. It seems to be associated with the option `--with-gslib`
See the logs for more info.

### CODE REVIEW

File: recipes/imagemagick/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    - fontconfig 2.12.*
+    - freetype 2.7
+    - ghostscript
+    - giflib 5.1.*
+    - graphviz
+    - jbig
+    - jpeg 9*
+    - libpng >=1.6.28,<1.7
+    - libtiff >=4.0.3,<4.0.8
+    - libwebp
```
Comment (bgruening) [inline recipes/imagemagick/meta.yaml]: Can we pin this as well? `libwebp 0.5.*`

**Comment by jdblischak** [comment]
As another data point, I was able to successfully build and install this recipe on my local Ubuntu 17.04 machine. I was able to run the code from the [ImageMagick Quick Start](https://www.imagemagick.org/download/QuickStart.txt) with no issues:

```
convert logo: logo.gif
display logo.gif
```

ImageMagick is a dependency for one of my recipes I'd like to submit, so please let me know if there is anything else I can do to test this recipe.

**Comment by ocefpaf** [comment]
@jdblischak I got side tracker with the day job and other stuff but I wanted to submit a few more dependencies that can be used here and a recipe for Windows.

However, I won't be able to get to that anytime soon. So let us merge this and improve it later on the feedstock.

File: recipes/imagemagick/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/imagemagick/meta.yaml]: We should probably open a PR to `pin_slow_way.py` with that.

**Pull request merged**